### PR TITLE
.net 7 support.

### PR DESCRIPTION
**Pull request by ReactiveThings**

### CODE REVIEW

File: Typezor.Tests/CodeModel/MethodTests.cs [hunk 1]
```diff
@@ -24,7 +31,7 @@ public void Expect_name_to_match_method_name()
 
             methodInfo.Name.ShouldEqual("Method");
             methodInfo.FullName.ShouldEqual("Typezor.Tests.CodeModel.Support.MethodInfo.Method");
-            methodInfo.Parent.ShouldEqual(classInfo);
+            //methodInfo.Parent.ShouldEqual(classInfo);
```
Comment (mukundkatpatal) [inline Typezor.Tests/CodeModel/MethodTests.cs:34]: May be missed to remove this commented code?

File: Typezor.Tests/CodeModel/MethodTests.cs [hunk 2]
```diff
@@ -56,9 +64,10 @@ public void Expect_to_find_parameters()
 
             methodInfo.Parameters.Count().ShouldEqual(1);
             parameterInfo.Name.ShouldEqual("parameter");
+            parameterInfo.FullName.ShouldEqual("string");
         }
 
-        [Fact]
+        
         public void Expect_to_find_parameter_attributes()
```
Comment (mukundkatpatal) [inline Typezor.Tests/CodeModel/MethodTests.cs:71]: any reason this is not a Fact?

**Review by mukundkatpatal — APPROVED** [review]
Approving with comments

File: Typezor.Tests/CodeModel/MethodTests.cs (hunk 1 above)
Comment (ReactiveThings) [reply Typezor.Tests/CodeModel/MethodTests.cs]: fixed

File: Typezor.Tests/CodeModel/MethodTests.cs (hunk 2 above)
Comment (ReactiveThings) [reply Typezor.Tests/CodeModel/MethodTests.cs]: fixed

**Review by dzerzi — APPROVED** [review]

**Pull request merged**